src/cascadia/WindowsTerminal/AppHost.cpp

```diff
@@ -1667,3 +1190,16 @@ void AppHost::_PropertyChangedHandler(const winrt::Windows::Foundation::IInspect
         }
     }
 }
+
+winrt::TerminalApp::TerminalWindow AppHost::Logic()

```

Comment:
i would have expected AppHost logic to return an AppLogic ;P

Reply:
Maybe this should be `WindowHost` now 😛

Reply:
can you imagine 
* `WindowEmperor`
* `WindowThread`
* `WindowHost`
* `TerminalWindow`

ew